server/sonar-server/src/test/java/org/sonar/server/ce/ws/CeWsModuleTest.java

```diff
@@ -30,6 +30,6 @@
   public void verify_count_of_added_components() {
     ComponentContainer container = new ComponentContainer();
     new CeWsModule().configure(container);
-    assertThat(container.size()).isEqualTo(11 + 2 /* injected by ComponentContainer */);
+    assertThat(container.size()).isEqualTo(12 + 2 /* injected by ComponentContainer */);
```

Comment:
FYI, `2` can be replaced by `ComponentContainer.COMPONENTS_IN_EMPTY_COMPONENT_CONTAINER`